lpa-submissions-e2e-tests/cypress/fixtures/anAppeal.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
an arbitrary bit of json grabbed from team1

this change basically makes all our tests inject "an appeal" before they start, and then uses the data from that appeal. in theory it would be good to have a few of these and test different conditions - that's now an option but i wasn't going to go there as part of this already massive job